README.md

```diff
@@ -744,6 +816,95 @@ The nearest grocery of SFU is Nesters Market. It is 1.234 miles far, and It is e
 
 </details>
 
+### Job
+
+#### [The Muse](./themuse) -- Collect Job Ads, Company Information
+
+<details>
+  <summary>What are the first 10 data science jobs I can do in Vancouver?</summary>
```

Comment:
What are the first 10 data science jobs I can do in Vancouver? -> What are 10 data science jobs in Vancouver?